FSI/Force.C

```diff
@@ -53,8 +53,10 @@ Foam::tmp<Foam::volSymmTensorField> preciceAdapter::FSI::Force::devRhoReff(dimen
 
 }
 
-void preciceAdapter::FSI::Force::write(double * buffer)
+void preciceAdapter::FSI::Force::write(double * buffer, bool provideMeshConnectivity)
 {
+    if(provideMeshConnectivity)
+        Info<<"Warning: Mesh connectivity is not supported for FSI cases"<<endl;
```

Comment:
A warning should be either a `adapterInfo(..., "warning")` or `WarningInFunction` instead of `Info`, with some note that this comes from the adapter.

Even further, since we don't want to get the same output in every time step, this warning should be implemented in the configuration phase.

Reply:
I would prefer to raise an error and abort here, since we actually skip the interpolation. The alternative would be to implement the interpolation here and allow it. 
What do you prefer?
If we don't implement it and just give a warning in the config we write wrong data.

Reply:
Then please add an error in the configuration and remove this warning. You may leave a `TODO` here.

Reply:
And open an issue for the non-implemented interpolations as well.

Reply:
Even if it is not necessary, since the solid part needs to provide the information?
Maybe we could assert in the adapter file, if the FSI enable is true and mesh connectivity?
Then, we don't have that much code duplication.

Reply:
Exactly, we need to check if both FSI and mesh connectivity are enabled.
The main goal is to avoid a late error (or a warning at every time step).

Reply:
Ok, then I will add a comment in each FSI related data files, that the error is raised in the adapter file. We don't need an implementation in each file. But you still want me to open up an issue, if I understand it correctly?

Reply:
If we don't implement some readers/writers now, but they could be useful in the future, then yes, please open an issue to keep track of the open tasks. :)